### PR TITLE
Modelica links

### PR DESCRIPTION
**Pull request by andreas-junghanns**
I had to rebase this to get the CI happy...

### CODE REVIEW

File: pages/about.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ ABB, DLR, ETAS, Fraunhofer (IIS/EAS First, SCAI), ITK Engineering, Open Modelica
 There are FMI tracks at other FMI related events like conferences and design meetings.
 If you believe you could benefit from the standard or if you would like to contribute to the improvement or distribution of it, please contact us at: [contact@fmi-standard.org](mailto:contact@fmi-standard.org).
 
-All individual contributors have to sign the [Corporate Contributor License Agreement (CCLA)](https://svn.fmi-standard.org/fmi/branches/public/FMI_CCLA_v1.0_2016_06_21.pdf) or must be corporate contributors that have signed the [CLA of the Modelica Association](https://www.modelica.org/licenses/ModelicaAssociationCLA_1.1).
+All individual contributors have to sign the [Corporate Contributor License Agreement (CCLA)](https://svn.fmi-standard.org/fmi/branches/public/FMI_CCLA_v1.0_2016_06_21.pdf) or must be corporate contributors that have signed the [CLA of the Modelica Association](https://modelica.org/licenses/ModelicaAssociationCLA_1.1.1.html).
```
Comment (chrbertsch) [inline pages/about.md:34]: https://svn.fmi-standard.org/fmi/branches/public/FMI_CCLA_v1.0_2016_06_21.pdf is no longer available

File: pages/literature.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ layout: default
 
 ### [13th International Modelica Conference 2019](https://2019.international.conference.modelica.org/)
 
-FMI User meeting ([Agenda](https://www.modelica.org/events/modelica2019/proceedings/html/userpres.html)):
+FMI User meeting ([Agenda](https://2019.international.conference.modelica.org/proceedings/html/userpres.html)):
 
 * [Overview of the FMI, DCP and SSP standards](https://svn.fmi-standard.org/fmi/branches/public/docs/Modelica2019/Industrial_User_Presentations_Modelica2019/001_Overview_FMI_SSP_DCP.pdf)
```
Comment (chrbertsch) [inline pages/literature.md:27]: all https://svn.fmi-standard.org links are broken

**Review by chrbertsch — CHANGES_REQUESTED** [review]

**Comment by chrbertsch** [comment]
Comment Copied from #305 
There are still some links broken to files that were prevously on SVN.
...

@t-sommer, @andreas-junghanns : Where shall we put the these files? To https://github.com/modelica/fmi-standard.org/tree/master/assets?

Do we have the PDFs any more? @MABackoffice : is there a backup of the SVN?

Some of the PPT files are here https://github.com/modelica/fmi-design/tree/master/Meetings/2019/2019-03-04-FMI-User-Meeting/Presentations but not all. 

We could also merge the PR and fix the remaining issues later

File: pages/about.md (hunk 1 above)
Comment (andreas-junghanns) [reply pages/about.md]: fixed

File: pages/literature.md (hunk 2 above)
Comment (andreas-junghanns) [reply pages/literature.md:27]: I think this should be done later - I am running into so many merge conflicts and have issues with tooling here, I would like to get this improvement in use and later fix more...

**Comment by andreas-junghanns** [comment]
If you agree, let´s merge ASAP and open another issue for the remaining issues and once we have a PR, quickly merge that before we get into this kind of trouble again.

**Comment by sjoelund** [comment]
> Do we have the PDFs any more? @MABackoffice : is there a backup of the SVN?

There is a backup available, but I'd need to bring it back online, figure out how to get the SVN server running, and then finally grab the files. It seems you found the files anyway.

**Review by chrbertsch — APPROVED** [review]
OK let's merge and fix the rest later

**Pull request merged**